mis_builder/models/aep.py

```diff
@@ -247,9 +247,9 @@ def get_aml_domain_for_expr(self, expr,
                 else:
                     continue
             if field == 'crd':
-                aml_domain.append(('credit', '>', 0))
+                aml_domain.append(('credit', '<>', 0.0))
             elif field == 'deb':
-                aml_domain.append(('debit', '>', 0))
+                aml_domain.append(('debit', '<>', 0.0))
```

Comment:
Is there a specific reason to change 0 to 0.0?

Otherwise I'm not opposed to the change.

Reply:
Not that matters a lot, but looks bit more pythonic, 
since expected field values are in float (0.0) , why compare it to integer (0) ...